.github/workflows/validate-solutions.yml

```
@@ -0,0 +1,168 @@
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'src/solutions/**'
+
+jobs:
+  validate:
+    runs-on: [self-hosted, Windows, X64]
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)